ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-base.yml

```diff
@@ -68,6 +68,13 @@ router_bgp:
       attribute_map: RM-ATTRIBUTE
       match_map: RM-MATCH
       advertise_only: true
+  redistribute_routes:
+    - source_protocol: ospf
+      include_leaked: true
+    - source_protocol: bgp
+      route_map: RM-REDISTRIBUTE-BGP
+      # this should not do anything as leaked is generated when source protocol is bgp
+      include_leaked: true
```

Comment:
we should remove it here and deprecate the key.

Reply:
The current model needs to support all routing protocols i.e ospf, bgp, static. etc., so we can't deprecate the inlcule_leaked feature.
My intent was to test to ensure that when configuring BGP, `include leaked` should not be generated.

CLI 

```eos
router bgp 65000
   redistribute ospf include leaked route-map ospf-bgp
   redistribute bgp leaked route-map bgp-bgp
   !
   vrf blah
      redistribute ospf include leaked route-map ospf-bgp
      redistribute bgp leaked route-map bgp-bgp
      !
      address-family ipv6
         redistribute bgp leaked route-map bgp-bgp
         redistribute ospfv3 include leaked route-map ospf-bgp
```

We should create a per-protocol data model in the future and deprecate this knob entirely, but as discussed, that is a lot of work and will be added to the backlog.